docker-files/launch-consul-config.sh

```diff
@@ -18,7 +18,12 @@
 
 set -e
 
-rm -rf /consul/data/* && $APP_DIR/docker-entrypoint.sh $CONSUL_ARGS | tee /edgex/logs/core-consul.log
+if [ -z $ARCH ]
+	then
+	rm -rf /consul/data/* && docker-entrypoint.sh agent $CONSUL_ARGS | tee /edgex/logs/core-consul.log
```

Comment:
Can you elaborate on what behavior you see here that requires the different syntax?

Reply:
In arm64, there is no "FROM consul" image and we have to manage it installing consul stuff manually, also script docker-entrypoint.sh, and to copy it into $APP_DIR. 

In "FROM consul" used for non-arm64, the script is in the path, so we cannot launch it from $APP_DIR but "path-less"